src/components/breadcrumbs/examples/BreadcrumbsExamples.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { Breadcrumb } from '../Breadcrumb';
+import { ITitleProps } from '../../title/Title';
+import { ILinkSvgProps } from '../../svg/LinkSvg';
+import { IBreadcrumbLinkProps } from '../BreadcrumbLink';
+
+export class BreadcrumbsExamples extends React.Component<any, any> {
+
+  render() {
+    const documentationLink: ILinkSvgProps = {
+      url: 'https://www.google.ca',
+      target: '_blank',
+      svg: {
+        svgName: 'help',
+        svgClass: 'fill-orange icon mod-20',
```

Comment:
as we discussed `mod-20` is not applied on the icon, you could remove

Reply:
I dont want to remove it, its important to be apply on the icon. its the right size for the icon to look good. I think its caused by a change in vapor

Reply:
the rule that is used in v2 is `.panel-header .coveo-icon-docs` could you consider implementing this one?

Reply:
nvm coveo-icon-docs is from v2 so does not make sens